### PR TITLE
Updating Getting started section

### PR DESCRIPTION
**Pull request by PtitBen**

### CODE REVIEW

File: docs/getting-started/working-with-nucleus.md [hunk 1]
```diff
@@ -14,10 +14,8 @@ description: How to work with the Nucleus Design System
...
-* Alternatively, download the [Sketch library](https://github.com/ConnectedHomes/centrica-ux/releases)
-* Take a look at the [Nucleus Playground](https://playground.nucleus.design) for an easy, visual way to prototype with the components.
-* Or if you are familiar with GitHub, you can install the Prototyping Tool on your local machine, check out the [Nucleus Prototyping Tool repository](https://github.com/britishgas-engineering/nucleus-prototype)!
+* For your ptototyping needs, take a look at the [Nucleus Playground](https://playground.nucleus.design) for an easy, visual way to prototype with the components. We have a [Playground support channel on Teams](https://teams.microsoft.com/l/channel/19%3a10af1f515b714d0cb7e30a8e8d0848c7%40thread.tacv2/%25F0%259F%258E%25A1%2520Playground%2520support?groupId=1f9c2411-216b-42a8-9bb0-c51f28ff5071&tenantId=a603898f-7de2-45ba-b67d-d35fb519b2cf) too! You will be able to take those prototypes into testing, in any browser and device, as well as testing them for accessibility using different assistive technology!
```
Comment (jholt1) [inline docs/getting-started/working-with-nucleus.md:17]: We don't want external links to the playground

Comment (jholt1) [reply docs/getting-started/working-with-nucleus.md:17]: How has this link been here for a whole year 😭 so many discussions about not publicly sharing the playground link

File: docs/getting-started/working-with-nucleus.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ description: How to work with the Nucleus Design System
...
-* For your ptototyping needs, take a look at the [Nucleus Playground](https://playground.nucleus.design) for an easy, visual way to prototype with the components. We have a [Playground support channel on Teams](https://teams.microsoft.com/l/channel/19%3a10af1f515b714d0cb7e30a8e8d0848c7%40thread.tacv2/%25F0%259F%258E%25A1%2520Playground%2520support?groupId=1f9c2411-216b-42a8-9bb0-c51f28ff5071&tenantId=a603898f-7de2-45ba-b67d-d35fb519b2cf) too! You will be able to take those prototypes into testing, in any browser and device, as well as testing them for accessibility using different assistive technology! 
+* For your ptototyping needs, take a look at the Nucleus Playground for an easy, visual way to prototype with the components. We have a [Playground support channel on Teams](https://teams.microsoft.com/l/channel/19%3a10af1f515b714d0cb7e30a8e8d0848c7%40thread.tacv2/%25F0%259F%258E%25A1%2520Playground%2520support?groupId=1f9c2411-216b-42a8-9bb0-c51f28ff5071&tenantId=a603898f-7de2-45ba-b67d-d35fb519b2cf) too! You will be able to take those prototypes into testing, in any browser and device, as well as testing them for accessibility using different assistive technology!
```
Comment (jholt1) [inline docs/getting-started/working-with-nucleus.md:17]: > ptototyping

prototyping

File: docs/getting-started/working-with-nucleus.md [hunk 3]
```diff
@@ -14,10 +14,8 @@ description: How to work with the Nucleus Design System
...
+* For your prototyping needs, take a look at the Nucleus Playground for an easy, visual way to prototype with the components. We have a [Playground support channel on Teams](https://teams.microsoft.com/l/channel/19%3a10af1f515b714d0cb7e30a8e8d0848c7%40thread.tacv2/%25F0%259F%258E%25A1%2520Playground%2520support?groupId=1f9c2411-216b-42a8-9bb0-c51f28ff5071&tenantId=a603898f-7de2-45ba-b67d-d35fb519b2cf) too! You will be able to take those prototypes into testing, in any browser and device, as well as testing them for accessibility using different assistive technology! 
+* British Gas design teams use Figma as their design tool. If you need to put a mockup together the Nucleus library is made available on our instance, and you should have access to the components. If you have any questions, we have [a dedicated Teams channel](https://teams.microsoft.com/l/channel/19%3ac806c8b36aec4218ae469b1d0ff6a4c3%40thread.tacv2/General?groupId=1f9c2411-216b-42a8-9bb0-c51f28ff5071&tenantId=a603898f-7de2-45ba-b67d-d35fb519b2cf).
```
[the remark this answers is not in the document]
Comment (RobTobias123) [reply docs/getting-started/working-with-nucleus.md:18]: The libraries are automatically available to anyone who joins the design teams at BG ... thinking that while this is a safe link, it's still a hint to where it is on a public document ? 

...

Perhaps then to be safe we could just mention that point instead?

[the remark this answers is not in the document]
Comment (RobTobias123) [reply docs/getting-started/working-with-nucleus.md:18]: Updated suggestion above to reflect this.

[the remark this answers is not in the document]
Comment (PtitBen) [reply docs/getting-started/working-with-nucleus.md]: Works for me. Go ahead and commit those suggestions and I'll be able to approve this PR :)

**Pull request merged**